data_demos/rnaseq-data-introduction.Rmd

```diff
@@ -0,0 +1,322 @@
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+    toc_float:
+      collapsed: false
+      smooth_scroll: false
+    fig_width: 7
+    fig_height: 6
+    fig_caption: true
+    df_print: paged  
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Getting Started
+
+This document aims to guide the challenge particpants with limited exposure to Synapse as a platform and clinical/biological data, through exploration and analysis of the available RNASeq datasets. The available dataset has been carefully curated to include gene expression (RNASeq) data from seven different tumor types from four different study initiatives into a single platform (a harmonized dataset). 
+
+The scope of this document is to inform participants :
+* how to navigate the Synapse environment to access and work with the data files
+* display the key features of the gene-expression (RNASeq) dataset 
+* introduce few general analyses steps of the data and their interpretation
+
+At the end of this document, the participants would be able to download the data and understand various intricacies that exists in this given dataset. We encourage participants to visualize the dataset by faceting on different variables to get an accurate understanding of the dataset. 
+
+## Loading Analytical Packages
+
+Lets first load the R packages we need to query, download, and analyze the data that is stored on Synapse.
+
+```{r lib_synapser, echo=FALSE, eval=TRUE, results='hide', message=FALSE, warning=FALSE}
+
+library(synapser)
+library(BiocManager)
+library(gProfileR)
+library(GOsummaries)
+library(tidyverse)
+
+```
+
+## Logging in to Synapse
+
+Please download your favorite Synapse client: [R](https://r-docs.synapse.org/) or [Python client](https://python-docs.synapse.org//build/html/index.html). Refer to the necessary docs [synapser docs](https://r-docs.synapse.org/articles/synapser.html#connecting-to-synapse) or [Python](https://python-docs.synapse.org//build/html/index.html) to successfully log in to synapse and start using its various analytical tools.
+
+```{r login_to_synapse, eval=TRUE, echo=TRUE}
+
+## The login credentials have been stored in .synapseConfig file in the home dir. If you dont have this setup, please make the file before trying to login.
+synLogin()
+
+```
+
+## Accessing or downloading data from Synapse
+
+* Look up the SynID of the required file and then use the __synGet__ command to access the folder. 
+* Download the required csv file by adding __$path__ to the accessed file
+
+```{r get_data_from_synapse, echo=TRUE, eval=TRUE}
+
+#### Download the harmonized file
+
+# Set file entity that can be changed easily for different files [User Input]
+fileEntity <- synGet('syn18137070')
+# Download the data file
+genes.with.meta<-read.csv(gzfile(fileEntity$path))
+```
+
+# Explore the data files
+
+## Structure of the file
+
+* The Structure of the file refers the column names and the type of variables stored in it:
```

Comment:
This might be prettier as a table - can you do that in Rmd?

Reply:
I can try to make a knitr table and see how it looks...